sdks/python/apache_beam/runners/interactive/testing/pipeline_assertion.py

```diff
@@ -0,0 +1,64 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+def assert_pipeline_equal(test_case, expected_pipeline, actual_pipeline):
+  """Asserts the equivalence between two given apache_beam.Pipeline instances.
+
+  Args:
+    test_case: (unittest.TestCase) the unittest testcase where it asserts.
+    expected_pipeline: (Pipeline) the pipeline instance expected.
+    actual_pipeline: (Pipeline) the actual pipeline instance to be asserted.
+  """
+  actual_pipeline_proto = actual_pipeline.to_runner_api(use_fake_coders=True)
+  expected_pipeline_proto = expected_pipeline.to_runner_api(
+      use_fake_coders=True)
+  components1 = actual_pipeline_proto.components
+  components2 = expected_pipeline_proto.components
+  test_case.assertEqual(len(components1.transforms),
+                        len(components2.transforms))
+  test_case.assertEqual(len(components1.pcollections),
+                        len(components2.pcollections))
+
+  # GreatEqual instead of Equal because the pipeline_proto_to_execute could
```

Comment:
This would mean that the assert_pipeline_equal(a, b) does not mean assert_pipeline_equal(b, a).
We should explain this in detail in the docstring of this function. Also, GreatEqual should be GreaterEqual.

Reply:
`assert_pipeline_equal(a, b)` should be symmetric.
The assertion code was copied from old pipeline_analyzer tests. We don't need the GreaterEqual any more. Changing the assertion statement and removing the comment.